int64[] curve
int64[] group
float64[] time_curve
float64[] time
float64[] pose2D_x
float64[] pose2D_y
float64[] pose2D_theta
int64[] mov_will
int64[] number
bool[] state
---
int64 to
int64 from